template_feed/Microsoft.DotNet.Test.Playwright.ProjectTemplates.8.0/content/MSTest-CSharp/UnitTest1.cs

```diff
@@ -0,0 +1,26 @@
+namespace Company.TestProject1;
+
+[TestClass]
+public class UnitTest1 : PageTest
+{
+    [TestMethod]
+    public async Task HomepageHasPlaywrightInTitleAndGetStartedLinkLinkingtoTheIntroPage()
```

Comment:
The fact that the template contains a fully working example on how to use Playwright's most important features is awesome. However, I was thinking that instead of having the template contain all this boilerplate code that will probably get deleted by the user anyways, we can have an empty test method with a comment, pointing to some MSDN link that contains this working example there. IIRC, the Selenium test templates do this.